scripts/check-public-api.ts

```diff
@@ -10,7 +10,7 @@ import {
   readCompilerOptions,
   transpileDirectory
 } from '@sap-cloud-sdk/generator-common/internal';
-import {defaultPrettierConfig} from "@sap-cloud-sdk/generator-common/dist/file-writer";
+import { defaultPrettierConfig } from '@sap-cloud-sdk/generator-common/dist/file-writer';
```

Comment:
[q] Could we avoid importing from dist and using the ts file?

Reply:
I guess so, I don't know why this was done this way originally (just formatted the file).

Reply:
`git blame`s you ;) I believe this was just overlooked in the review. Fixed it.

Reply:
Aaaaand, reverted it. When I remove `dist` it does not work. I didn't really check why, but I would suggest to separate this from this PR. I only changed this file, because apparently it is not covered by the formatter.